org/views.py

```diff
@@ -51,4 +52,41 @@ def post(self,request):
             return Response({}, status.HTTP_201_CREATED)
         else:
             data = serializer.errors
-            return Response(data, status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response(data, status.HTTP_400_BAD_REQUEST)
+
+
+@swagger_auto_schema(
+    operation_id="add_volunteer",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='get',
+    responses={
+        '200': set_example({"message": "You're added as a volunteer!"}),
+        '401': set_example({"detail": "Authentication credentials were not provided."})
```

Comment:
1. Please extract these responses to  `org.responses.py`
2. This response is not been documented 
![image](https://user-images.githubusercontent.com/33046846/82288352-e9e31c80-99bf-11ea-9d76-aacffb3e1c65.png)